assets/queries/crossplane/gcp/google_container_node_pool_auto_repair_disabled/metadata.json

```
@@ -0,0 +1,11 @@
+{
+  "id": "b4f65d13-a609-4dc1-af7c-63d2e08bffe9",
+  "queryName": "Google Container Node Pool Auto Repair Disabled",
+  "severity": "MEDIUM",
+  "category": "Insecure Configurations",
+  "descriptionText": "Verifies if Google Container Node Pool Auto Repair is Enabled. This service periodically checks for failing nodes and repairs them to ensure a smooth running state.",
```

Please, update the related queries

```suggestion
"descriptionText": "Google Container Node Pool Auto Repair should be enabled. This service periodically checks for failing nodes and repairs them to ensure a smooth running state.",
```